code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -350,12 +350,12 @@
 	item_state = "bullpup"
 	w_class = ITEMSIZE_LARGE
 	force = 10
-	caliber = "a556"
+	caliber = "a762"

```

Comment:
Missed a spot, but no, I do not have the sprites to replace the STS with the bullpup, nor do I particularly want to. The sprites have been changed to work as they do now.